### PR TITLE
super basic orgs page

### PR DESCRIPTION
**Pull request by UnsungHero97**
In spirit of keeping PRs small, this is a super basic, i.e. skeleton, implementation for the "home" / orgs page and the org details page.

Screen Shot 2019-07-22 at 11 19 29 AM

### CODE REVIEW

File: src/containers/orgs/OrgsContainer.js [hunk 1]
```diff
@@ -2,42 +2,153 @@
...
+    </Card>
+  )
+
+  render() {
+
+    const { getAllOrgsRequestState, organizations } = this.props;
+
+    if (getAllOrgsRequestState === RequestStates.PENDING) {
+      return (
+        <Spinner centered size="2x" />
```
Comment (solomontang) [inline src/containers/orgs/OrgsContainer.js]: I'm glad some of these props are working out for you, even if they're not super well documented.

File: src/containers/orgs/OrgsReducer.js [hunk 2]
```diff
@@ -65,39 +63,39 @@ export default function orgsReducer(state :Map<*, *> = INITIAL_STATE, action :Ob
...
-          const relevantEntitySets = {
-            appIdToEntitySetIdsMap: seqAction.value.appIdToEntitySetIdsMap
-          };
-          return state.set('relevantEntitySets', fromJS(relevantEntitySets));
-        },
-        FAILURE: () => state.set('relevantEntitySets', ENTITY_SETS_INITIAL_STATE),
-        FINALLY: () => state.set('isFetchingRelevantEntitySets', false),
-      });
-    }
+    // case getRelevantEntitySets.case(action.type): {
```
Comment (solomontang) [inline src/containers/orgs/OrgsReducer.js]: Still need this?

File: src/containers/orgs/components/OrgDescriptionSectionContainer.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      <>
+        <Description>{organization.get('description')}</Description>
+      </>
+    );
+  }
+}
+
+// $FlowFixMe
+export default withRouter(
+  connect()(OrgDescriptionSectionContainer)
```
Comment (solomontang) [inline src/containers/orgs/components/OrgDescriptionSectionContainer.js]: Empty connect?

File: src/containers/orgs/components/OrgTitleSectionContainer.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      <>
+        <Title>{organization.get('title')}</Title>
+      </>
+    );
+  }
+}
+
+// $FlowFixMe
+export default withRouter(
+  connect()(OrgTitleSectionContainer)
```
Comment (solomontang) [inline src/containers/orgs/components/OrgTitleSectionContainer.js]: Another empty connect?

File: src/containers/orgs/components/OrgOverviewContainer.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+      );
+    }
+
+    return (
+      <>
+        <OrgTitleSectionContainer organization={organization} />
+        <OrgDescriptionSectionContainer organization={organization} />
+      </>
+    );
+    // { this.renderDomainsSection() }
```
Comment (solomontang) [inline src/containers/orgs/components/OrgOverviewContainer.js]: Do you still need these?

File: src/containers/orgs/OrgsContainer.js (hunk 1 above)
Comment (UnsungHero97) [reply src/containers/orgs/OrgsContainer.js]: The spinner isn't centering vertically... needs more love.

**Pull request merged**